### DISCUSSION TITLE
createForm时副作用里有异步操作，表单提交后，表单内容会重置，怎样可以不重置

### DISCUSSION THREAD

**Discussion by ConanZZ** [alibaba/formily#1671] [Q&A 2.x] [answered]
场景：提交表单后，用户填入的值仍然填在表单中，而不是重置消失。请问有什么办法解决这个问题吗？

复现地址
https://codesandbox.io/s/wizardly-gareth-od047?file=/App.tsx

**Comment by janryWang** [comment, marked as answer]
低级问题，在渲染函数中创建form实例需要用useMemo包裹
